video/out/opengl/cuda_dynamic.c

```diff
@@ -27,7 +27,7 @@
 #endif
 
 #define CUDA_DECL(NAME, TYPE) \
-    TYPE *NAME;
+    TYPE *mpv_ ## NAME;
```

Comment:
There isn't really a need to rename all symbols - you could just define them. The define would use the native API function names, but would hide them from the linker. This would require adding an additional define for each function, so I'll leave it to you to decide, though.